spinetoolbox/spine_db_editor/mvcmodels/single_and_empty_model_mixins.py

```diff
@@ -32,7 +32,8 @@ def _convert_to_db(self, item):
         """
         item = item.copy()
         for field, real_field in self.field_map.items():
-            item[real_field] = item.pop(field, None)
+            if field in item:
+                item[real_field] = item.pop(field, None)
```

Comment:
Since we now know that `field` is in `item`, we don't need to specify the default parameter for `pop()` anymore, i.e.

```python
if field in item:
    item[real_field] = item.pop(field)
```

should suffice.